lib/lookup.json

```diff
@@ -543,7 +553,9 @@
   },
   "undici": {
     "prefix": "v",
-    "maintainers": ["mcollina", "ronag"]
+    "maintainers": ["mcollina", "ronag"],
+    "skip": ">=17",
+    "comment": "OpenSSL 3.0. https://github.com/nodejs/undici/issues/1085"
```

Comment:
I would prefer this to be fixed.

Reply:
This does not need to be skipped here, I think? It was updated to be skipped at the place where it fails in unidici itself? See https://github.com/nodejs/undici/commit/c5b7d03d9fbe958a6620c2ca33304f0a32be642a#diff-9f86d081884c7d659a2feaa0c55ad015a3bf4f1b2b0b822cd15d6c15b0f00a08

Reply:
This should be fixed.